### DISCUSSION TITLE
Always get error

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [shogo82148/actions-create-release#1031] [Q&A] [answered]
Hi,

I always get error:

<img width="1048" alt="image" src="https://github.com/shogo82148/actions-upload-release-asset/assets/395096/178bddf7-6596-431e-8efa-f066b12d9b2c">

Workflow:
https://github.com/paulocoutinhox/ai-sd-test/blob/main/.github/workflows/build.yml

Can you help me?

Thanks.

**Comment by shogo82148** [comment, marked as answer]
check the permission of your workflow please.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
the action requires the write contents permission.

```yaml
permissions:
  contents: write
```

Can I have the url of your workflow run?
I found https://github.com/paulocoutinhox/ai-sd-test/actions/runs/6679168959/job/18151071881, but it looks different from your screenshot.

<img width="1531" alt="image" src="https://github.com/shogo82148/actions-create-release/assets/1157344/5a575aa8-e5c4-4a4b-9d86-0e18baebaaf3">

**Comment by paulocoutinhox** [comment]
Thanks, i change the permission in my repository.
